packages/okta-react/README.md

```diff
@@ -352,76 +450,165 @@ class App extends Component {
 
 `SecureRoute` ensures that a route is only rendered if the user is authenticated. If the user is not authenticated, it calls `onAuthRequired` if it exists, otherwise, it redirects to Okta.
 
-### `ImplicitCallback`
+`SecureRoute` is specific to `react-router`.  Other routers will need their own methods to ensure authentication using the hooks/HOC props provided by this SDK.
```

Comment:
How about "was built specifically to integrate with `react-router`. If you are using a different router, you will need to..."

Reply:
> `SecureRoute` integrates with `react-router`.  Other routers will need their own methods to ensure authentication using the hooks/HOC props provided by this SDK.